ERCS/erc-7595.md

```diff
@@ -0,0 +1,491 @@
+---
+eip: 7595
+title: Collateralized NFT
+description: ERC-721 Extension to enable collateralization with ERC-20 based tokens.
+author: 571nKY (@571nKY), Cosmos (@Cosmos4k), f4t50 (@f4t50), Harpocrates (@harpocrates555)
+discussions-to: https://ethereum-magicians.org/t/collateralized-nft-standard/18097
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-13
+requires: 20, 721
+---
+
+## Abstract
+
+This proposal recommends an extension of [EIP-721](./eip-721.md) to allow for collateralization using a list of [EIP-20](./eip-20.md) based tokens. The proprietor of this EIP collection could hold both the native coin and [EIP-20](./eip-20.md) based tokens, with the tokenId acting as the access key to unlock the associated portion of the underlying [EIP-20](./eip-20.md) balance.
```

Comment:
Please don't forget to update all EIP-721, EIP-20 to ERC-721, ERC-20 etc

Reply:
All EIP updated for ERC.